### PR TITLE
Add deployment scale command

### PR DESCRIPTION
**Pull request by pigbug419**
Now we support deployment scale command in PF CLI!

### Note
`pf deployment <DEPLOYMENT_ID> <SCALE>` to scale deployment.

### CODE REVIEW

File: pfcli/deployment.py [hunk 1]
```diff
@@ -343,6 +341,7 @@ def create(
         "cloud": cloud,
         "region": region,
         "total_gpus": total_gpus,
+        "infrequest_perm_check": True,
```
Comment (kdh0102) [inline pfcli/deployment.py:344]: Is this argument always true?

File: pfcli/deployment.py [hunk 2]
```diff
@@ -355,6 +354,20 @@ def create(
     )
 
 
+@app.command()
+def scale(
```
Comment (kdh0102) [inline pfcli/deployment.py:358]: What do you think about let users set num_replicas when creating deployments?
Currently, two independent commands should be executed to launch multiple inferences servers in a deployment.

File: pfcli/service/__init__.py [hunk 3]
```diff
@@ -32,6 +32,7 @@ class ServiceType(str, Enum):
     GROUP_PROJECT_CHECKPOINT = "GROUP_PROJECT_CHECKPOINT"
     JOB_WS = "JOB_WS"
     DEPLOYMENT = "DEPLOYMENT"
+    DEPLOYMENT_SCALE = "DEPLOYMENT_SCALE"
     DEPLOYMENT_LOG = "DEPLOYMENT_LOG"
     DEPLOYMENT_METRICS = "DEPLOYMENT_METRICS"
     PFS_PROJECT_USAGE = "PFS_PROJECT_USAGE"
```
Comment (kdh0102) [inline pfcli/service/__init__.py:38]: So, we do not have to change any of these commands, right?

**Review by kdh0102 — CHANGES_REQUESTED** [review]
Nice work! A few comments.

File: pfcli/deployment.py (hunk 1 above)
Comment (pigbug419) [reply pfcli/deployment.py:344]: No, it's my fault to upload this line in this PR.
Thank you for pointing it out!

File: pfcli/deployment.py (hunk 2 above)
Comment (pigbug419) [reply pfcli/deployment.py:358]: Sounds good, I'll update create command.

File: pfcli/service/__init__.py (hunk 3 above)
Comment (pigbug419) [reply pfcli/service/__init__.py:38]: I already checked `usage` command, and it showed sum of all replicas' running time.
I'll check `metric` command! Thank you for the reminder.

**Comment by pigbug419** [comment]
Thank you for the review. I've addressed your comment!

**Pull request merged**